tests/test_sync_canary.py

```diff
@@ -1,35 +0,0 @@
-"""
```

Comment:
Any reason to remove these tests completely?

Reply:
Removed it because it was increasing execution time and not adding much value to the test suite.